web/src/components/task/task-sidebar/task-sidebar.tsx

```diff
@@ -238,43 +240,24 @@ const TaskSidebar: FC<TaskSidebarProps> = ({ onRedirectAfterFinish, jobSettings,
     }`;
     const validationStatus: ValidationPageStatus = isValid ? 'Valid Page' : 'Invalid Page';
 
-    // needed for taskSidebarLabel:
-    useEffect(() => {
-        refetchTaxons();
-    }, [latestLabelsId]);
+    const jobId = useMemo(() => getJobId(), [getJobId]);
```

Comment:
we already getting job from TaskAnnotator context, maybe we can use it here?

Reply:
yeah, it's it, look at line 100. It's just for not calling method getJobId() several times